portfolio/src/main/java/com/google/sps/ServerMain.java

```diff
@@ -13,7 +16,7 @@
  * classes annotated with the @WebServlet annotation.
  */
 public class ServerMain {
-
+  public static InitialValues first = new InitialValues();
```

Comment:
Might be worth giving it a more specific name, PageViewData

Reply:
Solved